### ISSUE TITLE
CICD  Pipeline hardening

### ISSUE THREAD

**Issue by nadgowdas** [OpenSecureSupplyChain/ossc#14]
What are the steps we need to take  to ensure security of our CICD pipelines.

**Comment by lukehinds** [comment]
How do you mean Shripad, such as hardening?

**Comment by nadgowdas** [comment]
Yes, "hardening" is probably the right characterization. I've changed the title to reflect that.  This would include controls like:
1. Pipeline signing : ensure all tasks/images are trusted
2. Configuration Checks: ensure best practices are followed in composing and configuring pipeline (e.g. running tasks are privilege, limiting access to shared resources etc.)
